lms/static/js/learner_dashboard/views/program_card_view.js

```diff
@@ -23,6 +24,25 @@
                 render: function() {
                     var templated = this.tpl(this.model.toJSON());
                     this.$el.html(templated);
+                    this.postRender();
+                },
+                postRender: function() {
+                    if(navigator.userAgent.indexOf('MSIE')!==-1
```

Comment:
Yeah, this is still needed. I tested IE10 and without picturefill, the images do not show